public/editor/scripts/main.js

```diff
@@ -79,5 +82,5 @@ require(["jquery", "bowser"], function($, bowser) {
     });
   }
 
-  require(["bramble-editor", "project/project", "sso-override", "fc/project-rename"], init);
+  require(["bramble-editor", "project/project", "sso-override", "fc/project-rename", "reloadProtection"], init);
```

Comment:
I defined it as reloadProtection in main.js above and used the same name below. Should it be unloadProtection? Or could you tell me what exactly to change?